integ-test/reports/report_2024-12-14-00.json

```diff
@@ -0,0 +1,1299 @@
+{
```

Comment:
Are all these test reports relevant to this PR?

Reply:
Not sure where they're from, it doesn't look like any other reports exist. I removed them. I wonder if they're supposed to be `.gitignore`d?